code/controllers/subsystem/processing/inbounds.dm

```diff
@@ -0,0 +1,35 @@
+var/datum/subsystem/objects/SSinbounds
```

Comment:
if I made it processing subsystem, I would have to put stuff in process() on all objects, and then put notes for people to call it if they wanted it to act like it, and it was messy.